index.html

```diff
@@ -151,6 +167,34 @@ <h1>Favorite Exercises</h1>
           >Add Exercises to Favorites</a
         >
       </section>
+      <section
+        class="new-workout-container flex flex-col align-center hidden"
+        data-view="new-workout">
+        <h1>Create a New Workout Plan</h1>
+        <form action="" class="new-workout-form flex flex-col">
+          <div class="form-items flex flex-col">
+            <label for="name" class="block">Name</label>
+            <input type="text" class="" id="name" />
```

Comment:
You should just wrap this `label` or `input` in a `div` to make it on it's own line. Changing an inline to a block element with CSS is generally bad practice because your content will probably appear meaningless to a user without CSS, such as a user needing vision assistance, or a search engine.